### PR TITLE
refactor(tensorrt_common)!: fix namespace, directory structure & move to perception namespace

### PR DESCRIPTION
**Pull request by amadeuszsz**
## Description

Change with respect to packages naming convention in autoware.universe & multi-container CI.

## Related links

**Parent Issue:**

- Link

## How was this PR tested?

```bash
ros2 bag play ~/autoware_data/rosbags/sample-rosbag/ -r 0.1 -p --loop
ros2 launch autoware_launch logging_simulator.launch.xml map_path:=$HOME/autoware_map/sample-map-rosbag vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit
```

## Notes for reviewers

- Namespaces in format which ensures compatibility with edge devices, please see https://github.com/autowarefoundation/autoware.universe/pull/8526 (cc: @manato). 
- PR is split by commits for [easier](https://github.com/autowarefoundation/autoware.universe/commit/da2c24ad149d6783f01db32dd272857c1bb464a8) changes tracking. Rename & replacement of package is separate [commit](https://github.com/autowarefoundation/autoware.universe/commit/679a784b81308c01517c68199ac27de9a5a1768d).

## Interface changes

- `tensorrt_common` package renamed to `autoware_tensorrt_common`
- project namespace changed from common to perception (cc: @youtalk)

## Effects on system behavior

None.

### CODE REVIEW

File: perception/autoware_tensorrt_yolox/CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.17)
 project(autoware_tensorrt_yolox)
 
-find_package(tensorrt_common)
-if(NOT ${tensorrt_common_FOUND})
+find_package(autoware_tensorrt_common)
+if(NOT ${autoware_tensorrt_common_FOUND})
```
Comment (knzo25) [inline perception/autoware_tensorrt_yolox/CMakeLists.txt:5]: nit: the warning still says `tensorrt_common` package

**Review by knzo25 — CHANGES_REQUESTED** [review]
Left a very small comment

File: perception/autoware_tensorrt_yolox/CMakeLists.txt (hunk 1 above)
Comment (knzo25) [reply perception/autoware_tensorrt_yolox/CMakeLists.txt:5]: (could not leave a comment on the actual line, since it was not a change on this PR)

Comment (amadeuszsz) [reply perception/autoware_tensorrt_yolox/CMakeLists.txt:5]: Ehh, I missed it. Thank you Kenzo!
Adressed in https://github.com/autowarefoundation/autoware.universe/pull/9099/commits/b3596304d19a8fd05377babe8fdf572bb24fa3b5.

**Review by knzo25 — APPROVED** [review]
LGTM

**Review by YoshiRi — APPROVED** [review]
LGTM

**Review by miursh — APPROVED** [review]

**Review by manato — APPROVED** [review]
@amadeuszsz 
LGTM. Thank you for your contribution!

**Pull request merged**